cosmos/cosmoshub-validator-delegations/subgraph.yaml

```diff
@@ -5,8 +5,8 @@ schema:
   file: ./schema.graphql
 dataSources:
   - kind: cosmos
-    name: {{ name }}
-    network: {{ network }}
+    name: cosmoshub-message-example
```

Comment:
Let's put the network name here (`CosmosHub`), to be consistent with other examples.